### DISCUSSION TITLE
PT助手白大增强版

### DISCUSSION THREAD

**Discussion by Rhilip** [ronggang/PT-Plugin-Plus#705] [Show and tell]
> 此处仅作代发，对于增强版本相关功能问题，请联系对应开发者。
> 此代发贴行为征得对应开发者同意。

来自 IYUU 2群的群友通过在options页面中外挂脚本的形式对PTPP进行了相关扩展，包括 新增等级升级显示、升级日历、延长cookie时间、一键签到、流量条 等功能。

## 安装

|类型|附件链接|
|---|---|
| 外挂源代码 | [白大ptpp插件-11(新增导入过期cookie功能).zip](https://github.com/ronggang/PT-Plugin-Plus/files/6036825/ptpp.-11.cookie.zip) <-- 点击下载 |
| 打包CRX | [白大插件PT-Plugin-Plus-v1.5.1集成版.crx.zip](https://github.com/ronggang/PT-Plugin-Plus/files/6036838/PT-Plugin-Plus-v1.5.1.crx.zip) <-- 点击下载 |

下载后均需要解压缩，如下载 `外挂源代码` 请参照相关使用说明进行安装。

## 相关功能截图 （来自 [百度贴吧](https://tieba.baidu.com/p/7222995761) ，表示感谢 ）：

![e54567d0f703918f4d12a883463d269758eec41f](https://user-images.githubusercontent.com/13842140/109018132-d012fa80-76f2-11eb-8466-d1a54b406273.png)
![e4b2f203918fa0ece604f144319759ee3c6ddb1f](https://user-images.githubusercontent.com/13842140/109018152-d4d7ae80-76f2-11eb-8a88-199155016be0.png)
![7161948fa0ec08fabec386ee4eee3d6d54fbda1f](https://user-images.githubusercontent.com/13842140/109018161-d7d29f00-76f2-11eb-88c3-7df1d17a5767.png)

## 其他

我们对相关代码实现进行过安全审查，确认过相关代码并没有可能造成危害的地方。

对于各位关心的 为什么不并入dev分支，目前主要有以下原因：
- 增强版的实现形式是以外挂script的形式，并不太符合vue开发编译的要求。
- 此外，增强版实现的一些功能是 PTPP 开发者所不太期望的，如 一键签到 （见 #13 ）

我们后续也会和增强版开发者保持联系，以便将部分实现的 feature 并入master/dev分支中。

**Comment by chinatengjie** [comment]
馒头域名改了  站点更改cdn 等级不显示 
尝试解压源码批量替换了域名  等级显示了 做种数又不显示了
能不能联系更新一下 或者其他方法

**Reply by jstxya666** [reply]
rank.js脚本里把mt的新域名替换后就正常了

**Comment by gzl09** [comment]
这个加强版的固定打包值是多少，用来加白名单。

**Comment by hecarli555** [comment]
1.5.2出来了，期待白大的新修改版

**Reply by jstxya666** [reply]
> 1.5.2出来了，期待白大的新修改版

用老的脚本套一样可以用，两个本来就是分开的东西，大白的只是一个外挂脚本而已

**Comment by hecarli555** [comment]
哦哦哦，好的那我试一试看，对了1.5.2版馒头的地址改过来了么？

**Reply by ap6689** [reply]
我自己测试了下，1.5.2是改过来了，不过外挂插件还没改，应该也是改地址即可正常

**Reply by ap6689** [reply]
修改外挂插件中的 rank.js 右键编辑，搜索到 team，把网址修正到正确地址，即可正确显示MT等级

**Comment by VicaYang** [comment]
1.5.2无痛升级 不过找安装目录按照教程没有成功，详情那里只显示了并非来自网上商店然后我没找到可以点击的地方
这里分享一下我找的方法
1. 浏览器输入 `chrome://version/` 可以显示个人资料路径（如我的win10下就是 `C:\Users\<YOUR_NAME>\AppData\Local\Google\Chrome\User Data\Default`)
2. 浏览器输入 `chrome://extensions/` 找到ptpp，会显示一个ID （我的是`dmmjlmbkigbgpnjfiimhlnbnmppjhpea`，不清楚是否都一样）
3. 进入1中的文件夹，然后依次进入`Extensions` `dmmjlmbkigbgpnjfiimhlnbnmppjhpea` `1.5.2_0`，此时就进入了插件的文件夹
4. 根据 word 教程的提示，把 `js` 和 `css` 放入对应的文件夹（事实上，直接把两个文件夹拖过来就好），并修改 `index.html`

**Comment by weigeloveu** [comment]
大佬们，edge怎么弄？修改之后就提示插件损坏

**Comment by AdoShan** [comment]
反馈一下，BtSchool显示签到成功实际并没有，此外HDTime以及HDFans都无法正常签到

**Reply by LD-KiLLer** [reply]
BtSchool是开了cloudflare墙，你在打开BtSchool的网页的同时再进行签到是可以正常的

**Comment by AdoShan** [comment]
BtSchool就没有其他办法吗？我目前浏览器直接输入网址是不需要cf认证

Sent from my iPhone

On Feb 8, 2022, at 11:08, LD-KiLLer ***@***.***> wrote:

﻿

BtSchool是开了cloudflare墙，你在打开BtSchool的网页的同时再进行签到是可以正常的

—
Reply to this email directly, view it on GitHub<https://github.com/ronggang/PT-Plugin-Plus/discussions/705#discussioncomment-2130190>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AKQ3Z66A4BIX5XIWWYD33MTU2CCK5ANCNFSM4YEVRLDQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by LD-KiLLer** [reply]
cloudflare的5秒盾现在暂时无解

**Reply by audioho** [reply]
请问下，部分网站升级数据更新了，如果要手动改，改哪个地方？

**Comment by usherkobe** [comment]


**Reply by ted423** [reply]
一键签到从来都不会被考虑

**Reply by usherkobe** [reply]
好的，我指的是这个插件，不过感谢解答

**Reply by ted423** [reply]
此处仅作代发，对于增强版本相关功能问题，请联系对应开发者。

**Reply by usherkobe** [reply]
好的
